query/logicalplan/logicalplan.go

```diff
@@ -27,6 +27,16 @@ type LogicalPlan struct {
 	Aggregation *Aggregation
 }
 
+// IterOptions are a set of options for the TableReader Iterators
+// TODO: should we instead use the option pattern? Is that possible with the way the Iterator functions work?
+type IterOptions struct {
+	PhysicalProjection []Expr
+	Projection         []Expr
+	Filter             Expr
+	DistinctColumns    []Expr
+	TimestampCol       string
```

Comment:
The `DistinctColumns` is fully written. Why not this column too?
```suggestion
	TimestampColumn       string
```